nav2_collision_monitor/launch/collision_monitor_node.launch.py

```diff
@@ -55,43 +60,69 @@ def generate_launch_description():
         default_value=os.path.join(package_dir, 'params', 'collision_monitor_params.yaml'),
         description='Full path to the ROS2 parameters file to use for all launched nodes')
 
-    # Create our own temporary YAML files that include substitutions
-    param_substitutions = {
```

Comment:
Is this not necessary anymore?

Reply:
It was used for `use_sim_time` substitution in a `params_file`. Since sim time was re-factored, params substitutions is not necessary anymore.